### PR TITLE
LPS-202232 Change Configuration view for Documents and Media

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-202232)

## Changes

Change the configuration for Documents and Media because it should be open in a new page instead of a modal.

## Notes

**You need to activate the LPS-197692 FF.**

## Test Scenario 1

1. Go to Documents and Media
2. Click on the kebab menu in the top right
3. Navigate to the sections to check the behaviour and styles

https://github.com/liferay-echo/liferay-portal/assets/93203423/d03b314e-090b-4916-94ee-dab2c414a74f

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library_admin/configuration_browse.jsp [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+		<clay:col
+			lg="9"
+		>
+			<h1>
+				<%= dlConfigurationDisplaycontext.getTitle() %>
+			</h1>
+
+			<liferay-portlet:actionURL portletConfiguration="<%= true %>" var="configurationActionURL">
+				<portlet:param name="navigation" value="<%= dlConfigurationDisplaycontext.getNavigation() %>" />
```
Comment (jkappler) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library_admin/configuration_browse.jsp:44]: @BarbaraCabrera, shouldn't this be liferay-portlet:param?

Comment (jkappler) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library_admin/configuration_browse.jsp:44]: based on the other params, seems it's different thatn in journal

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library_admin/configuration_browse.jsp [hunk 2]
```diff
@@ -29,5 +32,116 @@ portletDisplay.setURLBack(dlConfigurationDisplaycontext.getBackURL());
 				verticalNavItems="<%= dlConfigurationDisplaycontext.getNotificationsVerticalNavItemList() %>"
 			/>
 		</clay:col>
+
+		<clay:col
+			lg="9"
+		>
+			<h1>
+				<%= dlConfigurationDisplaycontext.getTitle() %>
+			</h1>
```
[the remark this answers is not in the document]
Comment (jkappler) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library_admin/configuration_browse.jsp:41]: @BarbaraCabrera, if we change this, we would need to change this in the other configuration views too (to standardize)

@p2kmgcl, nothing was mentioned in the first review, that's why it was left like this 😢

[the remark this answers is not in the document]
Comment (p2kmgcl) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library_admin/configuration_browse.jsp:41]: mmm maybe it is ok like that, but I am pretty sure that h1 is only used for the site name

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by jkappler** [comment]
@BarbaraCabrera, looks good to me from my side :D

@p2kmgcl, majo, feel free to forward it if you are okay too ❤️

**Comment by ealonso** [comment]
ci:stop

**Comment by ealonso** [comment]
@p2kmgcl @jkappler @BarbaraCabrera it should be review by Lima Team

**Pull request closed without merging**